src/v2/components/screens/HowItWorks/styles.ts

```diff
@@ -0,0 +1,137 @@
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+import { EasyMintLogo as _EasyMintLogo } from "../../EasyMintLogo";
+import { Whale as _Whale } from "../../Whale";
+
+export const Container = styled.article`
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  height: 100%;
+  width: 100%;
+  background: #1a1533;
+`;
+
+export const TitleWrapper = styled.div`
+  align-self: baseline;
+  margin-left: 48px;
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  width: 45%;
+`;
+
+export const Message = styled.div`
+  font-weight: 400;
+  font-size: 20px;
+  margin-top: 28px;
+  width: 60%;
+
+  @media (max-width: 940px) {
+    text-align: center;
+  }
+`;
+
+export const Title = styled.h1`
+  height: 56px;
+
+  /* Wally/H1 */
+
+  font-family: "Inter";
+  font-style: normal;
+  font-weight: 800;
+  font-size: 40px;
+  line-height: 56px;
+  /* identical to box height, or 140% */
+
+  letter-spacing: 0.0025em;
+  margin: 0.1em 0;
+  color: #f2f6f7;
+`;
+
+export const SubTitle = styled.div`
+  height: 64px;
+
+  font-family: "Inter";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  /* or 160% */
+
+  letter-spacing: 0.0025em;
+
+  color: #c0d0d8;
+`;
+
+export const BackButton = styled(Link)`
```

Comment:
could we add a hover effect to this like the button on the main landing page? could also be nice to make this into a secondary button component that is reusable